File: java/program/Snake.java
public class Snake extends Reptile {
    private String venomType;

    public Snake(String name, double length, String habitat, String diet, String venomType) {
        super(name, length, habitat, diet);
        this.venomType = venomType;
    }

    @Override
    public void display() {
        System.out.println("Snake: " + name + ", Venom: " + venomType +
                           ", Habitat: " + habitat);
    }
}

File: cpp/Program/Snake.cpp
#include "Snake.h"

Snake::Snake(string n, double l, string h, string d, string v)
    : Reptile(n, l, h, d), venomType(v) {}

void Snake::display() {
    cout << "Snake: " << name
         << ", Venom: " << venomType
         << ", Habitat: " << habitat << endl;
}
